Content.Server/Chemistry/ReagentEffects/WearableReaction.cs

```diff
@@ -0,0 +1,38 @@
+using Content.Shared.Inventory;
+using Content.Shared.Chemistry.Reagent;
+using JetBrains.Annotations;
+using Robust.Shared.Prototypes;
```

Comment:
Jetbrains?

Reply:
remove this and funny annotation

Reply:
Fairly certain the JetBrains stuff came from whatever ReagentEffect I used as a base for this one. Just leftover crap, should be removed.